lib/libutils/ext/include/asm_aarch64.S

```diff
@@ -0,0 +1,7 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (c) 2021, Linaro Limited
+ */
+
+#define BTI_C	bti c
+#define BTI_J	bti j
```

Comment:
I don't think it helps to define macros for these instructions, it's just obfuscation. Please use `bti c` and `bti j` directly.

Reply:
Sure, will change that